### PR TITLE
Added: slack notification on merged PRs

### PR DESCRIPTION
**Pull request by veghdev**
test notification with this pr

### CODE REVIEW

File: .github/workflows/slack.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    notify_merged_pr:
+        name: Notify on merged PR
+        runs-on: ubuntu-22.04
+        if: ${{ github.event.pull_request.merged == true }}
+        steps:
+            - uses: abinoda/slack-action@master
+              env:
+                  SLACK_BOT_TOKEN: ${{ secrets.VIZZUHQ_SLACK_API }}
+              with:
+                  args: '{\"channel\":\"${{ secrets.VIZZUHQ_SLACK_CHANNEL }}\",\"blocks\":[{\"type\":\"section\",\"text\":{\"type\":\"mrkdwn\",\"text\":\"*PR:* ${{ github.event.pull_request.title }}\n*Description :* ${{ github.event.pull_request.body }}\n*Merged at:* ${{ github.event.pull_request.merged_at }}\n<${{ github.event.pull_request.html_url }}|View PR>\"}}]}'
```
Comment (schaumb) [inline .github/workflows/slack.yml:17]: The "-s escapes are necessary?

Comment (schaumb) [inline .github/workflows/slack.yml:17]: Can the Author (+Reviewer) be added?

**Review by schaumb — APPROVED** [review]

File: .github/workflows/slack.yml (hunk 1 above)
Comment (veghdev) [reply .github/workflows/slack.yml]: unfortunately yes

Comment (veghdev) [reply .github/workflows/slack.yml]: I added the author, but users who approved the pr is not included in the event just through the api, but I don't think it's worth it.

**Review by schaumb — APPROVED** [review]

**Pull request merged**